tests/ipv6/run-ipv6-test.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+set -e # Exit immediately if a command exits with a non-zero status
+
+RELEASE=$1
+RELEASE_IMAGE=$2
+
+if [[ -z $RELEASE || -z $RELEASE_IMAGE ]]; then
+    echo "Usage: $0 <RELEASE> <RELEASE_IMAGE> [HELM_RELEASE_NAME] [NAMESPACE] [CLUSTER_NAME]"
+    echo "Error: RELEASE and RELEASE_IMAGE are required parameters. Example usage $(make ipv6-test RELEASE=vX.Y.Z RELEASE_IMAGE=release-X.Y-rc)"
+    exit 1
+fi
+
+HELM_RELEASE_NAME=${3:-ngf}
+NAMESPACE=${4:-nginx-gateway}
+CLUSTER_NAME=${5:-ipv6-only-${RELEASE}}
+RELEASE_REPO=ghcr.io/nginx/nginx-gateway-fabric
+
+cleanup() {
+    echo "Cleaning up resources..."
+    kubectl delete -f ipv6/manifests/ipv6-test-app.yaml || true
+    kubectl delete -f ipv6/manifests/ipv6-test-client.yaml || true
+    kubectl delete -f ipv6/manifests/gateway.yaml || true
+    helm uninstall ${HELM_RELEASE_NAME} -n ${NAMESPACE} || true
+    kind delete cluster --name ${CLUSTER_NAME} || true
+}
+
+trap cleanup EXIT
+
+kind create cluster --name ${CLUSTER_NAME} --config ipv6/config/kind-ipv6-only.yaml
+
+echo "Applying Gateway API CRDs"
+kubectl kustomize "https://github.com/nginx/nginx-gateway-fabric/config/crd/gateway-api/standard?ref=${RELEASE}" | kubectl apply -f -
+
+docker pull ${RELEASE_REPO}:${RELEASE_IMAGE}
+
+docker save ${RELEASE_REPO}:${RELEASE_IMAGE} | docker exec -i ${CLUSTER_NAME}-control-plane ctr --namespace=k8s.io images import -
```

Comment:
Is this even needed right now? If we're using the images that are published in Github, kind should just be able to pull those in automatically when a deployment is created.